tools/trap/ramdumpParser.py

```diff
@@ -772,12 +819,21 @@ def main():
 				ext = (line.split("=")[1].strip())
 	elf = (BIN_PATH + 'tinyara' + ext)
 
+	# Get the number of application binaries, names, text address and sizes
+	find_number_of_binaries(log_file)
+
 	print('')
 	print('')
 	print('*************************************************************')
-	print('dump_file         :', dump_file)
-	print('log_file          :', log_file)
-	print('elf_file          :', elf)
+	print('dump_file                   :', dump_file)
+	print('log_file                    :', log_file)
+	if (g_app_idx > 0):
+		print('Number of binaries          : %d [kernel + apps]' % (g_app_idx + 1))
+	else:
+		print('Number of binaries          : %d [kernel]' % (g_app_idx + 1))
```

Comment:
In this case, Number of binary is always 1.
So I think we can just print `1` directly, not calculating `g_app_idx` + 1.